src/global.json

```diff
@@ -1,4 +1,9 @@
 {
+	"sdk": {
+		"version": "7.0.302",
```

Comment:
Can we remove this and set global.json at root of the repository so it applies to the playground and test harness. We should also be able to remove the msbuild-sdks node too